Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1749,18 +1752,87 @@ namespace AtomSampleViewer
+
+                    // Find the appropiate format for the image
+                    for (uint32_t i = 0; i < static_cast<uint32_t>(RHI::Format::Count); ++i)
+                    {
+                        RHI::Format format = static_cast<RHI::Format>(i);
+                        RHI::FormatCapabilities capabilities = device->GetFormatCapabilities(format);
+                        if (RHI::CheckBitsAll(capabilities, RHI::FormatCapabilities::ShadingRate))
+                        {
+                            imageDescriptor.m_format = format;
+                            break;
+                        }
+                    }
+
+                    // Create the new asset with the proper size and format and register it in the RPI::AttachmentImage database
+                    RPI::AttachmentImageAssetCreator imageAssetCreator;
+                    imageAssetCreator.Begin(vrsImageAsset->GetId());
+                    imageAssetCreator.SetImageDescriptor(imageDescriptor);
+                    imageAssetCreator.SetName(vrsImageAsset->GetName(), vrsImageAsset->HasUniqueName());
+
+                    Data::Asset<RPI::AttachmentImageAsset> asset;
+                    if (imageAssetCreator.End(asset))
+                    {
+                        m_vrsTexture = RPI::AttachmentImage::FindOrCreate(asset);
+                        RHI::XRFoveatedLevel foveatedType = RHI::XRFoveatedLevel::None;
+                        // Check settings registry for the foveated level
+                        if (AZ::SettingsRegistryInterface* settingsRegistry = AZ::SettingsRegistry::Get())
+                        {
+                            if (u64 foveatedLevel; settingsRegistry->Get(foveatedLevel, RHI::XRFoveatedLevelKey))
+                            {
+                                AZ_Assert(
+                                    foveatedLevel <= static_cast<u64>(RHI::XRFoveatedLevel::High),
+                                    "Invalid foveated level %d", foveatedLevel);
+                                foveatedType = static_cast<RHI::XRFoveatedLevel>(foveatedLevel);
+                            }
+                        }
+                        // Fill up the contents of the shading rate image
+                        xrSystem->InitVariableRateShadingImageContent(m_vrsTexture->GetRHIImage(), foveatedType);                       
+                    }
+                }
+            }
+
             RPI::RenderPipelineDescriptor xrPipelineDesc;
             xrPipelineDesc.m_mainViewTagName = "MainCamera";
             xrPipelineDesc.m_renderSettings.m_multisampleState.m_samples = static_cast<uint16_t>(m_numMSAASamples);
 
             // Build the pipeline for left eye
             xrPipelineDesc.m_name = "RPISamplePipelineXRLeft";
-            xrPipelineDesc.m_rootPassTemplate = "LowEndPipelineXRLeftTemplate";
+            xrPipelineDesc.m_rootPassTemplate = "MultiViewXRLeftPipelineTemplate";
```

Comment:
Can you delete LowEndPipelineXRLeft.pass and LowEndPipelineXRRight.pass as well? No one should be using them anymore.